service/worker/archiver/util.go

```diff
@@ -98,6 +99,11 @@ func errorDetails(err error) string {
 	if _, ok := err.(*cadence.CustomError); !ok {
 		return details
 	}
-	err.(*cadence.CustomError).Details(&details)
+	// If no details can be extracted the Details function will return with an err no data message that we want to
```

Comment:
@yycptt  @andrewjdawson2016  can anyone of you take a look at this?

Reply:
I think the idea there is that, if the error has some details and the details can be parsed in to string, then return the details as string, otherwise just return an empty string.
How about changing the function to the following:
```
func errorDetails(err error) string {
    var details string
    customErr, ok := err.(*cadence.CustomError)
    if !ok {
        return ""
    }
    if err := customErr.Details(&details); err != nil {
        return ""
    }
    return details
}
```